sw/airborne/firmwares/rover/guidance/rover_guidance_steering.c

```diff
@@ -71,7 +72,7 @@ void rover_guidance_steering_init(void)
 void rover_guidance_steering_heading_ctrl(void)
 {
   float delta = 0.0;
-  float omega = guidance_control.gvf_omega; //GVF give us this omega
```

Comment:
It seems that the only dependency of this guidance control to GVF is this variable. Maybe it is worth passing this value as an argument of the function so it can be reused ?
The function itself is probably called from the autopilot XML file. It from here that you can decide to use GVF or another strategy.